src/app/app.component.html

```diff
@@ -120,49 +115,50 @@ <h2 class="nunito-normal what-are-oeb">
 						[routerLink]="['/signup']"
 						>{{ 'NavItems.register' | translate }}</a
 					>
-		
-					<div class="navitem" style="position: unset; display: flex; gap: 4px">
+
+					<!-- ToDo: comment in when all pages are completely translated -->
+					<!-- Language -->
+					<!-- <div class="navitem" style="position: unset; display: flex; gap: 4px">
 						<button (click)="changeLng('de')">
 							<span [ngStyle]="{ 'font-weight': selectedLng == 'de' ? 'bold' : 'normal' }">DE</span>
 						</button>
 						<span style="align-self: center">|</span>
 						<button (click)="changeLng('en')">
 							<span [ngStyle]="{ 'font-weight': selectedLng == 'en' ? 'bold' : 'normal' }">EN</span>
 						</button>
-					</div>
+					</div> -->
 				</div>
 
 				<!-- Login btn -->
-				<button
-					class="main-nav-bold login-btn"
-					[routerLink]="['/auth/login']"
-					>
-					Login
-				</button>
+				<button class="main-nav-bold login-btn" [routerLink]="['/auth/login']">Login</button>
 			</nav>
 		</ng-template>
 
 		<!-- Authenticated -->
 		<ng-template [ngIf]="loggedIn && !isOAuthAuthorizationInProcess">
 			<nav class="navbar-x-items" role="menubar">
 				<div class="navbar-x-navitems">
-					<a
-						role="menuitem"
+					<!-- About Badges Menu -->
+					<button
+						role="aboutBadges-menuitem"
+						aria-haspopup="true"
 						[class.navitem-light]="!theme.useColorNavbar"
-						class="navitem"
+						class="navitem u-cursor-pointer"
```

Comment:
I guess if we dont want to use styles from the screen.css file anymore, you could also use the ``` tw-cursor-pointer``` class